samples/Web/deploy-webapp-ip-restrictions/azurepolicy.json

```diff
@@ -0,0 +1,75 @@
+{
+    "properties": {
+        "displayName": "Web App default IP restrictions",
+        "mode": "indexed",
+        "description": "Configures Web App IP restriction to allow access from specified public IP addresses",
+        "metadata": {
+            "category": "Web"
+        },
+        "parameters": {},
+        "policyRule": {
+            "if": {
+                "field": "type",
+                "equals": "Microsoft.Web/sites"
+            },
+            "then": {
+                "effect": "DeployIfNotExists",
+                "details": {
+                    "type": "Microsoft.Web/sites/config",
```

Comment:
>type": "Microsoft.Web/sites/config", [](start = 21, length = 36)

you know you want to look at the "web" config so specifying the "name" here next to type will be more efficient

Reply:
Thanks.